packages/app/src/navigation/SidebarNavigation.vue

```diff
@@ -38,12 +38,12 @@
       >
         <RouterLink
           v-for="item in navigation"
-          v-slot="{ isActive }"
+          v-slot="{ isExactActive }"
           :key="item.name"
           :to="item.href"
         >
           <SidebarNavigationRow
-            :active="isActive"
+            :active="isExactActive"
```

Comment:
I opted for a global change instead of a Specs-only exception to ensure all navigation links continue to have the same behavior. 

This means that if any of the other pages have links to a child view with a different URL (whether it be query params, pathname addition, etc) the navigation link will lose its active styles. This is intentional so that the user can more intuitively understand how to return to the parent page.

Reply:
Yeah I agree with doing this as the default. It's likely in the future, if we have nested router views, there would be cases where the exact route changes but the user is still "on" the main page. But I'm fine with not worrying about that till we get to that point.